tests/tests_default_nm.yml

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- hosts: all
+  tasks:
+    - name: Set network provider to 'nm'
+      set_fact:
+        network_provider: nm
+    - name: Install NetworkManager
```

Comment:
This task is not needed here, it was only used to get the NM version in the other test.